IPython/html/templates/notebook.html

```diff
@@ -248,7 +248,7 @@
                         {% set
                             sections = (
                                 (
-                                    ("http://nbviewer.ipython.org/github/ipython/ipython/blob/2.x/examples/Index.ipynb", "Notebook Help", True),
+                                    ("http://nbviewer.ipython.org/github/ipython/ipython/blob/master/examples/Notebook/Index.ipynb", "Notebook Help", True),
```

Comment:
I just used the link specified by @ellisonbg in the issue discussion, seems like it would persist past the 3.0 release?